Makefile

```diff
@@ -1,13 +1,23 @@
+RUBY_VERSION := "$(shell ruby -v)"
+RUBY_VERSION_REQUIRED := "$(shell cat .ruby-version)"
+RUBY_MATCH := $(shell [[ "$(shell ruby -v)" =~ "ruby $(shell cat .ruby-version)" ]] && echo matched)
+
+.PHONY: ruby-version-check
+ruby-version-check:
+ifndef RUBY_MATCH
+	@printf "ruby $(RUBY_VERSION_REQUIRED) is required. Found %s\n" $(RUBY_VERSION)
```

Comment:
this will just print a warning, should we make the target fail? also should we just say how to install the correct version (e.g. `rbenv install 3.1.2`)